### DISCUSSION TITLE
Saving inconsistent data

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7769] [Help]
When I save my record, the first value is always correct but the 2 other sometimes contain the old value, instead of the new.
If I click the button again, then it saves the correct value.
I think the root case is that my values come from a data provider... Is this a bad practice ? 
<img width="678" alt="image" src="https://user-images.githubusercontent.com/28592069/190151619-dfe9fd9d-9420-41c9-8860-ab10fab8d67b.png">

**Comment by NicolasFabre81** [comment]
In fact the problem occurs when I change the value of my number field and then click validate.
If I change the value, go out of the field, then validate, it is always OK because the DP below did refresh correctly.
I of course added a refresh DP as the first action of my button, but it is not efficient.
<img width="247" alt="image" src="https://user-images.githubusercontent.com/28592069/190154735-97afa882-3385-471b-b7b7-3ead3294ca26.png">

**Comment by melohagan** [comment]
Hey @NicolasFabre81 

Would you be able to provide a screenshot of your component tree, and also outline the reproduction steps please?
